modules/asm/scripts/kubectl_wrapper.sh

```diff
@@ -21,33 +21,47 @@ if [ "$#" -lt 3 ]; then
     exit 1
 fi
 
-HOST=$1
-TOKEN=$2
-CA_CERTIFICATE=$3
+CLUSTER_NAME=$1
```

Comment:
This specifically looks like duplication of https://github.com/terraform-google-modules/terraform-google-gcloud/blob/master/modules/kubectl-wrapper/scripts/kubectl_wrapper.sh

Reply:
removed